apps/backoffice-v2/src/pages/Entities/hooks/useEntities/useEntities.tsx

```diff
@@ -20,15 +21,34 @@ export const useEntities = () => {
 
   const {
     meta: { totalPages },
-    data: cases,
+    data: _cases,
   } = data || { meta: { totalPages: 0 }, data: [] };
   const entity = useEntityType();
   const individualsSearchOptions = ['entity.name', 'entity.email'];
   const businessesSearchOptions = ['entity.name'];
   const { searched, onSearch, search } = useSearch({
-    data: cases,
+    data: _cases,
     searchBy: entity === 'individuals' ? individualsSearchOptions : businessesSearchOptions,
   });
+  const { data: users } = useUsersQuery();
+  const cases = useMemo(() => {
```

Comment:
Can you please explain the necessity of this section?

Reply:
Mapping users to cases as assignee to simplify way of accesing avatarUrl on render.

Reply:
The added code and complexity does not seem justified here. Note `_cases` make it seem like the variable is unused (due to convention).

Reply:
Are we not getting the assignee's avatar url from the db or endpoint?

Reply:
We do now,updated

Reply:
Do we still need a map if the avatarUrl is on `assignee` itself?